### DISCUSSION TITLE
crosh corrupt command

### DISCUSSION THREAD

**Discussion by POGGRS** [3kh0/ext-remover#320] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i need the code to corrupt the extensions on crosh back . Just paste the crosh command in the comments if any of u guys has the command

**Comment by Alex2Mayhem** [comment, marked as answer]
> i need the code to corrupt the extensions on crosh back . Just paste the crosh command in the comments if any of u guys has the command

What crosh commands? There is no commands to turn off extensions.

**Reply by POGGRS** [reply]
i remember using it but 3kh0 removed the redictory link that said "try this method" which led to a crosh command to corrupt it.

**Reply by ehunt34** [reply]
He removed it because of me. It revolved around a bash exploit. https://bugs.chromium.org/p/chromium/issues/detail?id=1329945

**Reply by POGGRS** [reply]
thanks! i can  do the rest on crosh

**Comment by Aka-but-nice** [comment]
If there was a command chrome would be sued, tbh
